NEWS

```diff
@@ -1,3 +1,51 @@
+Overview of changes leading to 3.0.0
+Friday, September 17, 2021
+====================================
+- The hb-subset API of the harfbuzz-subset is now stable.
```

Comment:
Is it “harfbuzz-subset” or “the harfbuzz-subset”?

Reply:
```suggestion
- The hb-subset API of harfbuzz-subset is now stable.
```
I read this several times, my English must be getting worse because David is right...

Reply:
Should be: "of the harfbuzz-subset library".